index.d.ts

```diff
@@ -0,0 +1,20 @@
+/**
+Regular expression for matching [astral symbols](https://everything2.com/title/astral+plane).
+
+@param exact - Only match an exact string. Useful with `RegExp#test()` to check if a string is a astral symbol. Default: `false` _(Matches any astral symbols in a string)_
```

Comment:
Just curious, any technical reason you're using `_` instead of `*` (like in the readme) for italic?

Reply:
It conflicts with doc comment processing at least in VSCode. VSCode thinks it's the beginning of doc comment line.

Reply:
I think we should consider this as relevent, a lot of people use VSCode and doc comments are very useful for them.

Reply:
Sure, I agree. I was just curious.